docs/src/modules/components/HomePro.js

```diff
@@ -28,14 +27,15 @@ const styles = theme => ({
 
 function HomePro(props) {
   const { classes } = props;
+  const t = useSelector(state => state.options.t);
 
   return (
     <Link
       variant="body2"
       className={classNames(classes.root, 'mui-fixed')}
       href="/getting-started/support/#professional-support-premium"
     >
-      Get Professional Support
+      {t('getProfessionalSupport')}
```

Comment:
I guess this wasn't translated because we don't offer support in most of these languages. But we can for German at least (I happen to know a little bit :smile:) and I think on the offchance someone messages due to this message we'll might be able to work around the language barrier. Just to get a foot in the door.

Reply:
Correct, it was done under the following assumptions 1. Enterprise, in this context, speaks English. No objection to the change. At least, I don't see the harm in exploring. Could you translate it in Crowdin afterward? :)